defs.h

```diff
@@ -120,39 +122,37 @@
 #define SCREEN_BPP		32
 #endif
 
-#define BORDER_LEFT		64
-#define BORDER_TOP		160
-#define BORDER_RIGHT	64
-#define BORDER_BOTTOM	16
+#define BORDER_LEFT		(SCREEN_WIDTH / 10)

```

Comment:
The border is actually a fixed size. But by doing this you also had to decompose all the numbers that incorporate it, meaning the prework for removing the border is now done. So not waisted work :)